source/_data/themes.yml

```diff
@@ -3330,3 +3330,15 @@
     - modern
     - elegant
     - 中文
+
+- name: Dark
+  description: A different hexo theme 
+  link: https://confuseder.github.io/ 
+  preview: https://github.com/confuseder/hexo-dark-theme
```

Comment:
Would you mind renaming this theme to `hexo-theme-dark` ?
`hexo-dark-theme` will be considered to be a plugin rather than a theme by Hexo. Thanks.

https://github.com/hexojs/hexo/blob/463025c9d21411570e56b15812eb319ea0883330/lib/hexo/load_plugins.js#L41

Reply:
is ok

Reply:
so i need to change the name to hexo-theme-dark